app/src/forms/common/middleware/validateParameter.js

```diff
@@ -93,12 +93,19 @@ const validateExternalAPIId = async (req, _res, next, externalAPIId) => {
     _validateUuid(externalAPIId, 'externalAPIId');
 
     const externalApi = await externalApiService.readExternalAPI(externalAPIId);
-    if (!externalApi || externalApi.formId !== req.params.formId) {
+    if (!externalApi) {
```

Comment:
On no - I broke the admin route, didn't I? I didn't notice that it doesn't have a :formId on it.

Reply:
i thought it was me! just going to kick it on openshift and test before merge.